compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -217,7 +217,6 @@ struct ConvertToROCDLPass final
       populateLowerHALInterfaceOp(llvmPatterns);
       populateLLVMConversionPatterns(&getContext(), llvmPatterns, converter);
       populateComplexToLLVMConversionPatterns(converter, llvmPatterns);
-      populateMathToLLVMConversionPatterns(converter, llvmPatterns);
```

Comment:
You won't want to get rid of this since this has patterns that the device libraries don't - that is, MathToROCDL only includes things that don't get covered by MathToLLVM